### PR TITLE
pincode_request_is_not_starting_when_rotating

### PR DESCRIPTION
**Pull request by purigarcia**

### CODE REVIEW

File: src/com/owncloud/android/ui/activity/PassCodeActivity.java [hunk 1]
```diff
@@ -96,13 +98,23 @@ protected void onCreate(Bundle savedInstanceState) {
...
-            setCancelButtonEnabled(true);
+            if (savedInstanceState != null) {
+                mConfirmingPassCode = savedInstanceState.getBoolean(PassCodeActivity.KEY_CONFIRMING_PASSCODE);
+                mPassCodeDigits = savedInstanceState.getStringArray(PassCodeActivity.KEY_PASSCODE_DIGITS);
+                if(mConfirmingPassCode){
+                    //the app was in the passcodeconfirmation
+                    requestPassCodeConfirmation();
+                }
+            }
+            if(savedInstanceState == null || (savedInstanceState != null && !mConfirmingPassCode)){
```
Comment (davivel) [inline src/com/owncloud/android/ui/activity/PassCodeActivity.java]: This condition is unnecessary complex. Enough with :

```if (!mCofirmingPassCode)```

**Comment by davivel** [comment]
Reviewed; please, @purigarcia , check my comment.

File: src/com/owncloud/android/ui/activity/PassCodeActivity.java (hunk 1 above)
Comment (purigarcia) [inline src/com/owncloud/android/ui/activity/PassCodeActivity.java]: Ok. Thanks for the review

**Comment by davivel** [comment]
:+1:

**Pull request merged**